specification/columns/commitmentdiscountpurchasequantity.md

```diff
@@ -0,0 +1,35 @@
+# Commitment Discount Purchase Quantity
+
+Commitment Discount Purchase Quantity is the numeric value of [CommitmentDiscountUnits](#commitmentdiscountunit) that is eligible for amortization for any full or partial upfront purchase of a [*commitment-based discount*](#glossary:commitment-based-discount).
+
+CommitmentDiscountPurchaseQuantity MUST be present in the billing data when the provider supports *commitment-based discounts*. CommitmentDiscountPurchaseQuantity value MUST be the total amount of units, denoted by CommitmentDiscountUnit, available for a *commitment-based discount* for its [*term*](#glossary:term) at the time of an applicable purchase.
+
+CommitmentDiscountPurchaseQuantity MUST be a positive value when [ChargeCategory](#chargecategory) is *Purchase*, [PricingCategory](#pricingcategory) is *Committed*, and [CommitmentDiscountId](#commitmentdiscountid) is not null when a purchase record exists for a *commitment-based discount*.
```

Comment:
"when a purchase record exists for a *commitment-based discount*." - seems we woud leave the door open here for a provider to add the colun because they support CDs (though above we describe its for full or partial up-front only) and in the 3rd paragraph, further weaken the language to 'when a purchase record exists'

Reply:
Is the suggestion to remove or alter?